src/pages/Design/DesignPage.vue

```diff
@@ -0,0 +1,127 @@
+<template>
```

Comment:
@ryan-ludwig This page should really be using the ContentfulPage with a new PageType that can then use it's own Page Frame.

This will enable usage of all available Contentful components for adding content.

Your 2 new components below should be made available in the Contentful page.

Reply:
In Contentful you'd update the "Page" Content Model, Adding a new Validation option to the "Page Type" field.